python/fbprophet/forecaster.py

```diff
@@ -1474,3 +1524,58 @@ def copy(self, cutoff=None):
             DeprecationWarning,
         )
         return prophet_copy(m=self, cutoff=cutoff)
+
+    def make_holidays(self, year_list, country_list):
```

Comment:
let's make this not a class method since this file is already way too long. This can be moved to a separate file and just imported here, since it doesn't actually use `self` at all.